app/views/people/index.html.haml

```diff
@@ -1,7 +1,8 @@
 = render partial: "people/search", locals: { person: nil }
 - flash.each do |type, msg|
-  .alert.alert-danger.mt-3
-    = msg
+  - if type == 'error'
```

Comment:
Well now we don't show any messages besides `error` anymore :thinking: 
Is that really the idea?